subsys/net/openthread/Kconfig.defconfig

```
@@ -242,4 +242,13 @@ config OPENTHREAD_BLE_TCAT_THREAD_STACK_SIZE
 
 endif # OPENTHREAD_BLE_TCAT
 
+config OPENTHREAD_NUM_MESSAGE_BUFFERS
+	int "The number of message buffers in the buffer pool"
+	range 96 65535 if OPENTHREAD_FTD
+	range 64 65535
+	default 96 if OPENTHREAD_FTD
+	default 64
+	help
+	  "The number of message buffers in the buffer pool."
+
```

```suggestion
config OPENTHREAD_NUM_MESSAGE_BUFFERS
	range 96 65535 if OPENTHREAD_FTD
	range 64 65535
	default 96 if OPENTHREAD_FTD
	default 64
```
As this kconfig is already defined in zephyr, we shoul only override range and default value